src/ati/terraform.py

```diff
@@ -110,7 +115,7 @@ def inner(*args, **kwargs):
 
 
 def _parse_prefix(source, prefix, sep='.'):
-    for compkey, value in source.items():
+    for compkey, value in list(source.items()):
```

Comment:
Another list wrapper... is this a compatibility thing? This should work on python2 and 3 just fine without it.